lib/partition_setup.pm

```diff
@@ -268,11 +268,11 @@ sub addlv {
     send_key $cmd{system_view};
     send_key 'home';
     send_key_until_needlematch('volume_management_feature', 'down');
-    wait_still_screen 2;
+    wait_still_screen(stilltime => 3, timeout => 4);
     # Expand collapsed list with VGs
     send_key 'right' if is_sle('<15');
```

Comment:
Perhaps `send_key_until_needlematch "needle_covering_subitem_colapsed", 'right'` could help here? avoiding also going down for more clarity where it fails.

Reply:
Actually, default timeout is 30 seconds, so it's then about still time?

Reply:
I guess it could possibly be the case, since 2 seconds were not enough for YaST2.  

```
[2019-01-30T17:44:04.0755 CET] [debug] /var/lib/openqa/cache/tests/sle/tests/installation/partitioning_full_lvm.pm:28 called partition_setup::addlv
[2019-01-30T17:44:04.0756 CET] [debug] <<< testapi::wait_still_screen(timeout=30, stilltime=2, similarity_level=47)
[2019-01-30T17:44:06.0808 CET] [debug] >>> testapi::wait_still_screen: detected same image for 2 seconds, last detected similarity is 1000000
[2019-01-30T17:44:06.0809 CET] [debug] /var/lib/openqa/cache/tests/sle/tests/installation/partitioning_full_lvm.pm:28 called partition_setup::addlv
[2019-01-30T17:44:06.0810 CET] [debug] <<< testapi::send_key(key='right', do_wait=0)
```

Reply:
Anyways, it doesn't hurt, so let's try it.